index.html

```diff
@@ -1,121 +1,164 @@
+              best they can be at any field they wish to be in.
             </div>
-        </section>
-        <section>
-            <div class="grid">
-                <div>
-                    <h1><u>WHAT WE DO</u></h1>
-                    <p>
-                        Absolute Robotics competes in the annual FIRST Robotics Competition (FRC), an international
-                        competition where teams design and build an industrial-sized robot within 6 weeks to participate
-                        in an engineering challenge. Teams compete with robots in their local and international
-                        community at competitions around the world, predominantly in Ontario and the USA.
-                    </p>
-                </div>
-                <img src="/assets/images/lp/showcase.jpg">
+          </div>
+        </div>
+      </div>
+      <div id="what-we-do" class="section">
+        <h2 class="section-title">What We Do</h2>
+        <div class="flex flex-col gap-32 text-lg">
+          <div class="two-column">
+            <div class="paragraph">
+              <div class="subparagraph">
+                Absolute Robotics is a robotics team based at the Woodlands
+                Secondary School in Mississauga, Ontario. We are currently made
+                up of over 70 students, teachers, and mentors who are passionate
+                about entrepreneurship, business, engineering, and technology!
+              </div>
+              <div class="subparagraph">
+                Our members gain invaluable hands-on experiences in the world of
+                engineering, problem solving and business. By competing in the
+                FIRST Robotics Competition (FRC), they develop key skills and
+                competences that are crucial in our constantly evolving
+                technological society.
+              </div>
             </div>
-        </section>
+            <img
+              alt="Absolute Robotics team members posing with their robot."
+              class="rounded-lg image-beside-paragraph"
+              src="assets/images/index/showcase.jpg"
+            />
+          </div>
+        </div>
+      </div>
     </main>
 
-    <hr class="zbm">
-    <footer style="text-align: center;">
-        <a href="https://www.facebook.com/frc4308" style="color: #3b5998"><i class="bx bxl-facebook-square bx-lg"></i></a>
-        <a href="https://twitter.com/frcteam4308" style="color: #1da1f2"><i class="bx bxl-twitter bx-lg"></i></a>
-        <a href="https://www.youtube.com/channel/UCFOy9P3ZDVzclLQF3w-mtsw" style="color: #ff0000"><i class="bx bxl-youtube bx-lg"></i></a>
-        <a href="https://www.instagram.com/frc4308/" style="color: #c13584"><i class="bx bxl-instagram bx-lg"></i></a>
-        <a href="mailto:the.woodlands.robotics@gmail.com"><i class="bx bx-mail-send bx-lg"></i></a>
+    <footer class="footer">
+      <div>Made with ♡ by Team 4308 & vidhan</div>
```

Comment:
Can this entire line be removed as it looks unprofessional.

Reply:
Fixed with 639d8ce56fb62db6c4aa404b63b6c8e5a9a17bba.